### PR TITLE
Fix liveserver process termination

### PR DESCRIPTION
**Pull request by jadkik**
Fixes issue pytest-dev/pytest-flask#49.

### CODE REVIEW

**Comment by jantman** [comment]
I'd be very interested in seeing this merged, as coverage would be very helpful...

**Comment by vitalk** [comment]
I'm so sorry for a long delay and trying do my best, so the all pending PRs will be merged within a week.

**Comment by nicoddemus** [comment]
Hey @vitalk no worries, your effort is appreciated by all the pytest community! 😄

**Comment by jantman** [comment]
@vitalk I'm sorry, I tried to phrase that as passively as I could, but I could've done better. I know what it's like to get "hey bro, merge this" comments, and I should've been more mindful.

I just wanted to "ping" on this since it's gone a few months with no comments. "It will get looked at one day, but I haven't had time yet" is a perfectly acceptable answer (not that an answer is even required).

**Comment by vitalk** [comment]
This is my bad, it's easy to miss some things when time goes by.

**Comment by galuszkak** [comment]
@vitalk will You merge this anytime soon? Appreciate Your hard work on pytest-flask 👍

**Comment by RazerM** [comment]
If I can help in any way, I'd like to see this merged.

**Comment by nicoddemus** [comment]
@RazerM thanks for the offer.

After #81 gets merged and `0.11` is released, I plan to review and merge some of the open PRs and make another `0.12` release.

File: tests/test_live_server.py [hunk 1]
```diff
@@ -58,6 +58,53 @@ def test_a(live_server):
...
+                def index():
+                    return 'got it', 200
+
+                live_server.start()
+
+                res = urlopen(url_for('index', _external=True))
+                assert res.code == 200
+                assert b'got it' in res.read()
+        ''')
+        result_with = appdir.runpytest('-v',
```
Comment (nicoddemus) [inline tests/test_live_server.py]: I think depending on coverage here is a little too brittle. 

I see a few options:
* monkeypatching `LiveServer.stop` to ensure it was called at the end of the session instead. 
* Use a "verbose" mode which prints to the output that it has been cleanly shutdown, then check the output by passing `-s` to pytest.

**Comment by nicoddemus** [comment]
Thanks @jadkik for the PR and sorry for the delay!

**Pull request merged**